zipline/pipeline/loaders/blaze/core.py

```diff
@@ -952,6 +954,17 @@ def collect_expr(e):
                 columns=added_query_fields + list(map(getname, columns)),
             )
         )
+        if not materialized_deltas.empty and have_sids:
+            missing_sids = materialized_deltas[
+                ~materialized_deltas.sid.isin(assets)
+            ]
+            log.debug(
+                "Didn't find the following sids in asset index: {}".format(
```

Comment:
The databazaar is only one consumer of this, users may come with their own datasets that have some intersection with the assets that exist in their asset finder but also contains others. Also, this is relying on the implementation detail that the assets requested is currently the full universe in existance. There are plans to make this filter to only sids that have matched some other filters so I am not totally sure this logging at this particular spot is desired for all users of zipline